app/assets/stylesheets/application.scss

```diff
@@ -65,3 +65,19 @@ $pagination-link-btn-border-color: #ddd;
     }
   }
 }
+
+.profile-head h5{
+    color: #333;
+}
+
+
+.tab-content profile-tab{
+  margin: 2%;
+}
+
+.col-md-8{
```

Comment:
Do not override default bootstrap classes unless absolute necessary. Use margin classes provided by bootstrap if you want to add margin. Or create your custom class.

Reply:
ok